app/assets/stylesheets/plugins/wice_grid.scss

```diff
@@ -1,148 +1,81 @@
-.wg-detached-filter,
-.wice-grid {
+.wg-detached-filter, .wice-grid{
+
   .text-filter-container {
-    input {
-      display: inline;
+    input{
+      width:auto;
       margin-right: 10px;
-      width: auto;
+      display: inline;
     }
   }
 
-  a {
-    .date-label {
-      text-decoration: none;
-    }
-
-    .date-label:hover {
-      text-decoration: line-through;
-    }
+  thead th select {
+    display: inline-block;
   }
 
-  .clickable {
+  a.date-label {text-decoration: none;}
+  a.date-label:hover {text-decoration: line-through;}
+
+  .clickable{
     cursor: pointer;
     margin-bottom: 2px;
     margin-right: 2px;
   }
 
-  .ui-datepicker-trigger,
-  .wg-detached-filter .ui-datepicker-trigger {
+  .ui-datepicker-trigger, .wg-detached-filter .ui-datepicker-trigger{
     cursor: pointer;
   }
 
   .custom-dropdown-container {
-    .expand-multi-select-icon,
-    .collapse-multi-select-icon {
-      display: inline-block;
+
+    .expand-multi-select-icon, .collapse-multi-select-icon{
+      width: 10px;
       height: 10px;
+      display: inline-block;
       margin-left: 5px;
       vertical-align: top;
-      width: 10px;
-    }
-
-    .collapse-multi-select-icon {
-      background: transparent asset-url("/assets/icons/grid/collapse.gif") no-repeat;
-    }
-
-    .expand-multi-select-icon {
-      background: transparent asset-url("/assets/icons/grid/expand.gif") no-repeat;
     }
   }
 }
 
-.wice-grid {
-  .desc,
-  .asc {
-    padding-right: 18px;
-    text-decoration: none;
-  }
-
-  .desc {
-    background: transparent asset-url("/assets/icons/grid/arrow_down.gif") right no-repeati;
-  }
-
-  .asc {
-    background: transparent asset-url("/assets/icons/grid/arrow_up.gif") right no-repeat;
-  }
+.wice-grid{
 
-  .submit.clickable {
-    @include icon-dimensions;
-    background: transparent asset-url("/assets/icons/grid/table_refresh.png") no-repeat;
-  }
-
-  .reset.clickable {
-    @include icon-dimensions;
-    background: transparent asset-url("/assets/icons/grid/table.png") no-repeat;
-  }
-
-  .wg-show-filter.clickable,
-  .wg-hide-filter.clickable {
-    @include icon-dimensions;
-    background: transparent asset-url("/assets/icons/grid/page_white_find.png") no-repeat;
-  }
-
-  .export-to-csv-button.clickable {
-    @include icon-dimensions;
-    background: transparent asset-url("/assets/icons/grid/page_white_excel.png") no-repeat;
-  }
-
-  .clickable.select-all {
-    @include icon-dimensions;
-    background: transparent asset-url("/assets/icons/grid/tick_all.png") no-repeat;
-    float: left;
+  .desc,  .asc{
+    padding-right: 18px;
+    text-decoration:none;
   }
 
-  .clickable.deselect-all {
-    @include icon-dimensions;
-    background: transparent asset-url("/assets/icons/grid/untick_all.png") no-repeat;
-    float: left;
+  .clickable.select-all, .clickable.deselect-all{
```

Comment:
Each selector in a comma sequence should be on its own single line<br>Opening curly brace `{` should be preceded by one space